radix-blueprint-schema-init/src/lib.rs

```diff
@@ -113,6 +113,15 @@ pub enum TypeRef<T> {
                // TODO: How to represent a structure containing a generic?
 }
 
+impl<T> TypeRef<T> {
+    pub fn into_static(self) -> T {
```

Comment:
Maybe we return an `Option<T>` and do the unwrap at client side?

Reply:
Yeah seems reasonable to me, will do that. I kinda want to add another method that panics, perhaps:

* `into_static`: returns an option.
* `assert_static`: returns T and panics if it's not static.